app/src/main/java/eu/kanade/tachiyomi/data/download/anime/AnimeDownloader.kt

```diff
@@ -517,6 +524,15 @@ class AnimeDownloader(
         }
     }
 
+    private fun isTor(video: Video): Boolean {
+        return when {
+            video.videoUrl?.startsWith("magnet") == true -> true
+            video.videoUrl?.endsWith(".torrent") == true -> true
+            video.videoUrl?.startsWith(TorrentServerUtils.hostUrl) == true -> true
+            else -> false
+        }
+    }
```

Comment:
```suggestion
    private fun isTorrent(video: Video): Boolean {
        return when {
            video.videoUrl?.startsWith("magnet") -> true
            video.videoUrl?.endsWith(".torrent") -> true
            video.videoUrl?.startsWith(TorrentServerUtils.hostUrl) -> true
            else -> false
        }
    }
```

Reply:
why not make it like this then
```kt
private fun isTorrent(video: Video): Boolean {
        return videoUrl?.startsWith("magnet") || videoUrl?.endsWith(".torrent") || videoUrl?.startsWith(TorrentServerUtils.hostUrl)
}
```

Reply:
aaa fuck videourl can be null thats why is == true